libs/common/src/schemas/audit-database-record.test.ts

```diff
@@ -0,0 +1,48 @@
+import { ObjectId } from 'mongodb';
+import {
+  generateNoUserLoggedInAuditDatabaseRecord,
+  generatePortalUserAuditDatabaseRecord,
+  generateSystemAuditDatabaseRecord,
+  generateTfmUserAuditDatabaseRecord,
+} from '../change-stream';
+import { withSchemaValidationTests } from './with-schema-validation.tests';
+import { AUDIT_DATABASE_RECORD } from './audit-database-record';
+
+describe('AUDIT_DATABASE_RECORD', () => {
+  const A_VALID_AUDIT_RECORD = generateTfmUserAuditDatabaseRecord(new ObjectId());
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { lastUpdatedAt, ...A_AUDIT_RECORD_WITH_MISSING_PARAMETER } = { ...A_VALID_AUDIT_RECORD };
+  const A_AUDIT_RECORD_WITH_INCORRECT_PARAMETER_TYPE = { ...A_VALID_AUDIT_RECORD, lastUpdatedAt: 1 };
+
+  const successTestCases = [
+    {
+      description: 'a valid audit record',
```

Comment:
Think these descriptions should be more specific  (e.g. a valid TFM audit record)